docs/guide/types/line.md

```diff
@@ -70,7 +70,14 @@ The following options are available for line annotations. All of these options c
 
 ### General
 
-If one of the axes does not match an axis in the chart, the box will take the entire chart dimension. The 4 coordinates, xMin, xMax, yMin, yMax are optional. If not specified, the box is expanded out to the edges in the respective direction.
+If one of the axes does not match an axis in the chart and the line behaviors are the following, depending how the line should be drawn:
+
+1. if `scaleId` is not resolved, the line will take the entire chart dimension, starting top-left vertex to bottom-right vertex of the chart
+1. if `xScaleId` is not resolved, the line will take the entire chart width
+1. if `yScaleId` is  not resolved, the line will take the entire chart height
```

Comment:
Probably a mistake with the numbers?

Reply:
Maybe I'm wrong but it works (MD syntax). Do you prefer to have 1, 2, 3, directly in MD file?

![lineDoc](https://user-images.githubusercontent.com/11741250/143558002-b3a5d580-07c8-45d2-b01f-1a999e672364.png)